src/models/DeploymentScheduleCompat.ts

```diff
@@ -0,0 +1,6 @@
+import { Schedule } from '@/models/Schedule'
+
+export type DeploymentScheduleCompat = {
```

Comment:
nit: we've generally tried to keep away from truncated/abbreviated variable names in these codebases